apps/server/src/modules/lesson/service/lesson-copy.service.ts

```diff
@@ -223,18 +224,17 @@ export class LessonCopyService {
 				copiedContent.push(linkContent);
 				copiedContentStatus.push(embeddedTaskStatus);
 			}
-			if (element.component === ComponentType.NEXBOARD && nexboardEnabled) {
-				// eslint-disable-next-line no-await-in-loop
+
+			if (element.component === ComponentType.NEXBOARD && nexboardEnabled && copyNexboardEnabled) {
```

Comment:
so, if copy feature flag is false, nothing is returned.
if copy is true, a NOT_DOING status is returned. 
this is not how it should work...

if copy flag is false, then it should return status NOT_DOING
if it is true, it should remain as it is...